flang/lib/Lower/OpenMP.cpp

```diff
@@ -178,14 +179,38 @@ genOMP(Fortran::lower::AbstractConverter &converter,
                          &clause.u)) {
         const Fortran::parser::OmpObjectList &ompObjectList = copyinClause->v;
         genObjectList(ompObjectList, converter, copyinClauseOperands);
+      } else if (const auto &allocateClause =
+                     std::get_if<Fortran::parser::OmpAllocateClause>(
+                         &clause.u)) {
+        mlir::Value allocatorOperand;
+        const Fortran::parser::OmpObjectList &ompObjectList =
+            std::get<Fortran::parser::OmpObjectList>(allocateClause->t);
+        /// Check if allocate clause has allocator specified. If so, add it
```

Comment:
NIT: No need for Doxygen style comments ?
Use C++ style instead '//' . And Finish it with period.

Reply:
I see, I'll change this right away. Thanks for the review